app/lib/account/models.dart

```diff
@@ -129,6 +129,9 @@ class UserSession extends db.ExpandoModel {
   @db.StringProperty(required: true)
   String email;
 
+  @db.StringProperty()
+  String name;
```

Comment:
Can we give this a comment, saying what it is? It's not a unique name, right? Can it be null?